.drone.yml

```diff
@@ -619,10 +619,7 @@ steps:
       commit: true
       commit_message: "[skip ci] Updated translations via Crowdin"
       remote: "git@github.com:go-gitea/gitea.git"
-    environment:
```

Comment:
Just replace `appleboy/drone-git-push` with `appleboy/drone-git-push:linux-amd64`

Reply:
@6543 I will update the latest image with multiple platforms.